TACC/Core-CMS
# ⚠️ test(core-styles): tup-274 peerDeps not deps

**⚠️ Do not merge. This is only for testing. ⚠️**

## Overview

Test Core-Styles' `task/tup-274-core-styles-deps` before merging relevant PR.

## Related

- [TUP-274]
- testing https://github.com/TACC/Core-Styles/pull/46

## Changes

- install Core-Styles at `task/tup-274-core-styles-deps`

## Testing

1. Build styles for any CMS project e.g.
    - `npm run build --project=core-cms`
    - `npm run build --project=frontera-cms`
2. Verify the app builds without error.[^1][^2]

[^1]: A CMS stylesheet build error caused by a missing dependency will halt the build.
[^2]: An error caused by a missing stylesheet will **not** halt the build (known problem). Search for `Error: Failed to find `.

[TUP-274]: https://jira.tacc.utexas.edu/browse/TUP-274
